### PR TITLE
Typescript SDK changes and bug fixes needed for Connect

### PR DESCRIPTION
**Pull request by kev1n-peters**
- Added a WormholeTransferStandardRelayer payload type needed to parse the VAA for an automatic transfer between EVM chains
- EVM: getIsExecuted should check if the transfer is inbound queued to determine if the transfer is completed
- Solana: getIsExecuted should check if the inboxItem is released
- Solana: getIsApproved should not swallow exception
- Solana: getInboundQueuedTransfer should return null if the transfer is not queued
- Bumped SDK dependencies needed for Connect

### CODE REVIEW

**Comment by kev1n-peters** [comment]
Should probably release SDK `0.8.0` and change the dependencies in here to `^0.8`

File: evm/ts/src/ntt.ts [hunk 1]
```diff
@@ -143,19 +141,30 @@ export class EvmNtt<N extends Network, C extends EvmChains>
...
+        : attestation.payload.payload;
+    const isExecuted = await this.manager.isMessageExecuted(
+      Ntt.messageDigest(attestation.emitterChain, payload["nttManagerPayload"])
     );
+    if (!isExecuted) return false;
+    // Also check that the transfer is not queued for it to be considered complete
+    const queued = await this.getInboundQueuedTransfer(
+      attestation.emitterChain,
+      payload["nttManagerPayload"]
+    );
```
Comment (barnjamin) [inline evm/ts/src/ntt.ts:157]: Should we have a separate `isQueued` fn on the interface?

Comment (kev1n-peters) [reply evm/ts/src/ntt.ts:157]: Maybe it would make the code cleaner, I can add that

File: sdk/route/src/types.ts [hunk 2]
```diff
@@ -153,4 +167,15 @@ export namespace NttRoute {
...
   }
+
+  // returns true if the amount is greater than 95% of the capacity
+  // useful for warning about the possibility of a transfer being queued
+  export function isCapacityThresholdExceeded(
+    amount: bigint,
+    capacity: bigint
+  ): boolean {
+    const threshold = (capacity * 95n) / 100n;
+    console.log(amount, threshold, amount > threshold);
```
Comment (barnjamin) [inline sdk/route/src/types.ts:178]:
```suggestion
```

File: solana/ts/sdk/ntt.ts [hunk 3]
```diff
@@ -543,16 +545,26 @@ export class SolanaNtt<N extends Network, C extends SolanaChains>
...
-    if (!this.getIsApproved(attestation)) return false;
-
-    const { emitterChain } = attestation as WormholeNttTransceiver.VAA;
-    const inboundQueued = await this.getInboundQueuedTransfer(
-      emitterChain,
-      attestation
-    );
+  async getRateLimitDuration(): Promise<bigint> {
+    // The rate limit duration is hardcoded to 24 hours on Solana
+    return BigInt(24 * 60 * 60);
```
Comment (barnjamin) [inline solana/ts/sdk/ntt.ts:550]: always?

**Review by barnjamin — DISMISSED** [review]

File: solana/ts/sdk/ntt.ts (hunk 3 above)
Comment (kev1n-peters) [reply solana/ts/sdk/ntt.ts:550]: This is hard-coded in the contract and there's no method we can call like we do to get the version

**Review by barnjamin — DISMISSED** [review]

**Review by barnjamin — APPROVED** [review]

File: sdk/route/src/manual.ts [hunk 4]
```diff
@@ -125,24 +130,55 @@ export class NttManualRoute<N extends Network>
...
+      const dstAmount = amount.parse(
+        params.amount,
+        request.destination.decimals
+      );
+      if (
+        NttRoute.isCapacityThresholdExceeded(amount.units(dstAmount), capacity)
+      ) {
+        result.warnings = [
+          {
+            type: "DestinationCapacityWarning",
```
Comment (artursapek) [inline sdk/route/src/manual.ts:165]: extremely minor comment, this could be a shared const. but im fine with it the way it is

**Review by artursapek — APPROVED** [review]

File: sdk/route/src/manual.ts (hunk 4 above)
Comment (kev1n-peters) [reply sdk/route/src/manual.ts:165]: agreed, will revisit when we add more warnings. don't want to make changes now to dismiss approvals

**Pull request merged**